bool contactSafe   # /**< Whether we got contact with ground or not. */
bool contact
bool noContactSafe  # /** Whether we got no contact with ground or not */
